### DISCUSSION TITLE
Support for .srt subtitles

### DISCUSSION THREAD

**Discussion by nithinvisva** [cookpete/react-player#1797] [Ideas]
.srt subtitles are not appearing in player.

**Comment by Fyb3roptik** [comment]
This will depend on if your client supports them. Many only support WebVTT

**Comment by AntonOfTheWoods** [comment]
This would probably not be too difficult to support at least simple conversion from one format to another, depending on what is supported. There are plenty of libs out the that do an ok job, even if the features of the various formats aren't exactly the same.
